examples/kitchensink-ts/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "kitchensink-ts",
+  "private": true,
+  "version": "4.1.1",
+  "description": "A Strapi application.",
+  "scripts": {
+    "develop": "strapi develop",
+    "develop:ce": "STRAPI_DISABLE_EE=true strapi develop",
+    "start": "strapi start",
+    "build": "strapi build",
+    "build:ce": "STRAPI_DISABLE_EE=true strapi build",
+    "strapi": "strapi"
+  },
+  "dependencies": {
+    "@strapi/admin": "4.1.1",
+    "@strapi/provider-email-mailgun": "4.1.1",
```

Comment:
Do you think about adding the i18n and the users-permissions plugins so we don't have to build the admin panel when starting the server as it will use the default build?

Reply:
The need to be generated inside the dist folder anyway so I'm not sure it'll make a difference here

Reply:
Well we are building the admin if the default plugins are not installed (or if we have other plugins) or if we have some customisations.